recipes-devtools/syslinux/syslinux_6.%.bbappend

```diff
@@ -1,5 +1,6 @@
-PACKAGES =+ "${PN}-isohybrid"
-PACKAGES =+ "${PN}-mboot"
+PACKAGES =+ " \
+	     ${PN}-isohybrid \"
+             ${PN}-mboot \"
```

Comment:
Bitbake will not parse this:
```
ERROR: ParseError at [...]xenclient-oe/recipes-devtools/syslinux/syslinux_6.%.bbappend:3: unparsed line: '             ${PN}-mboot \"'
```
```PACKAGES =+``` does not have an ending double-quote and there is an escaped one in between which should not be here. The snippet provided earlier was:
```sh
PACKAGES =+ " \
    ${PN}-isohybrid \
    ${PN}-mboot \
"
```